plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""The entropy calculation implementation."""
+
+from __future__ import division
+from __future__ import unicode_literals
+
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
+  """Calculates the byte entropy of input files."""
+
+  NAME = 'entropy'
+  DESCRIPTION = 'Calculates the entropy of input data.'
```

Comment:
entropy => byte entropy

since you're calculating the entropy in the byte value distribution

Reply:
Done